specifications/20200316-enabling-experimental-features.org

```diff
@@ -0,0 +1,62 @@
+#+TITLE: 20200316 Enabling Experimental Features
+#+AUTHOR: Sergio Schvezov <sergio.schvezov@canonical.com>
+#+DATE: [2020-03-15]
+
+* Scope
+It is often required to release a feature for Snapcraft *hidden* behind flags to
+not inadvertently make use of them without tending to the consequences for its
+potential instability, the instability can be present in different ways:
+
+- The feature itself is not stable or does not consider all the corner cases.
+- The command line interface is subject to change.
+- The command output is subject to change.
+
+This document defines a common pattern to follow to enable features that must be
+disabled by default.
+* Guidelines
```

Comment:
While it renders fine, can you [set your editor to] include empty line breaks between empty sections?  Just makes the raw text more readable without having a proper viewer. :-)